packages/.tsconfig.packages.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
What is the difference between this and the tsconfig in the root of the project? Should we just have the root tsconfig and the sub-tsconfigs point to that?